.editorconfig

```diff
@@ -16,5 +13,19 @@ fsharp_max_array_or_list_width=120
 fsharp_max_infix_operator_expression=80
 fsharp_max_value_binding_width=120
 
+# Specific settings for "View/HTML" related files
+# It makes the code more consistent in term of spacing and indentation
 [GenerateHtml.fs]
-fsharp_experimental_elmish = true
\ No newline at end of file
+fsharp_experimental_elmish = true
+fsharp_array_or_list_multiline_formatter = number_of_items
+fsharp_max_array_or_list_number_of_items = 0
```

Comment:
I don't really like this one, it makes too many single item list format in three lines. Makes it all rather excessive.

Reply:
This makes the code more consistent it always align to the 4 spaces guides and avoid having to reformat the list over and over when adding a new element:

![CleanShot 2025-01-02 at 11 10 52@2x](https://github.com/user-attachments/assets/d36a7c11-1834-4d0f-814a-d568de9c076d)

For example, if I want to add a new element to `embed` with that configuration I can simply press `Enter` and be ready with it. Without I, need to move the list, align it, split the inner value on multiple lines, etc. Or add `;` to add the new item but if it is too long I need to go back manually placing stuff around.

Reply:
pressing `;` and formatting isn't the end of the world.
How many times are really facing this issue? If you had this once I really don't feel like this is worth it. If you have this 20 times in an hour I could kinda get it, but even then...

Reply:
More often than I want to have to deal with it 🙂

Reply:
Removing this code also make it harder to read some line IHMO.

![CleanShot 2025-01-02 at 20 14 26@2x](https://github.com/user-attachments/assets/639ec8b3-51f8-4d73-a8f4-2d034b9de4b4)

I removed it but we can re-introduce if wanted before merging.